include/CppUTestExt/MockNamedValue.h

```diff
@@ -85,7 +85,6 @@ class MockNamedValue
     virtual void setObjectPointer(const SimpleString& type, const void* objectPtr);
     virtual void setSize(size_t size);
 
-    virtual void setComparator(MockNamedValueComparator* comparator);
```

Comment:
It is never used. Comparaters are always set via installComparator().

If I understand what this is for, I can write a test for it and not remove it ;-)